### PR TITLE
rogue update hook

### LINKED ISSUE #7016: Create rogue tracking tables
## _FEATURE OVERVIEW_

Create two tables `dosomething_rogue_reportbacks` and `dosomething_rogue_reportback_items` 

`dosomething_rogue_reportbacks` will store a reporbacks `rbid` in drupal and it's corresponding `rogue_reportback_id`

`dosomething_rogue_reportback_items` will store `rbid`, `fid`, `rogue_item_id` 

These tables will be used so that phoenix can identify reportbacks in rogue that are stored with different ids.

### PR DESCRIPTION
**Pull request by sbsmith86**
#### What's this PR do?

Creates an update hook to install the new rogue table retroactively.
#### Relevant tickets

Fixes #7016 
#### Checklist
- [ ] Tested on staging.

### CODE REVIEW

File: lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install [hunk 1]
```diff
@@ -49,3 +49,12 @@ function dosomething_rogue_schema() {
 
   return $schema;
 }
+
+/**
+ * Create failed task log table.
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install]: this should just be the rogue reportback table 🎐

Comment (sbsmith86) [reply lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install]: ack copy paste fail

Comment (sbsmith86) [reply lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install]: @angaither fixed!

**Comment by angaither** [comment]
📜 🔙

**Pull request merged**